backend/src/main/java/com/java/academy/gym/model/Locale.java

```diff
@@ -0,0 +1,76 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "locales")
+public class Locale extends BaseEntity {
```

Comment:
Prefer a natural key to a surrogate key if the natural key is just one column. Here you can create the natural key for example using the "languageCode" field